lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -24,6 +24,12 @@ const String transparentTextEditingClass = 'transparentTextEditing';
 
 void _emptyCallback(dynamic _) {}
 
+/// The root node that hosts all DOM required for text editing.
```

Comment:
This is not accurate. When a11y is enabled we use `<input>` elements created by the semantics tree. We could describe this as "default" text editing root (I'd also rename the getter to `defaultTextEditingRoot`).

Reply:
Hm, note that the semantic tree also renders inside the ShadowDOM, so this is still technically true! (however, I rephrased :P)